examples/seat-adjuster/src/main.py

```diff
@@ -19,7 +19,7 @@
 import logging
 import signal
 
-from sdv_model import Vehicle, vehicle  # type: ignore
+from vehicle import Vehicle, vehicle  # type: ignore
```

Comment:
shouldn't this stay from sdv_model? like in all other examples.

Reply:
Changed that to be "importable" in template ...